hooks/useWhiteList.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from "react"
+import { whitelistedAddresses } from "@/lib/whitelistAddresses"
+import useConnectedWallet from "@/hooks/useConnectedWallet"
+const useWhitelist = () => {
+  const { externalWallet } = useConnectedWallet()
+
+  useEffect(() => {
+    checkIfWhitelisted()
+  }, [externalWallet])
+
+  const checkIfWhitelisted = () => {
+    const address = externalWallet?.address
+    const isWhitelistedAddress = whitelistedAddresses.includes(address)
+    const istrue = isWhitelistedAddress
+    return istrue
+  }
```

Comment:
move this to a lib `lib/checkIsWhitelisted`

should remove the need for a hook altogether, right?